frontend/src/queries/nodes/WebOverview/WebOverview.tsx

```diff
@@ -24,28 +23,20 @@ export function WebOverview(props: {
     const logic = dataNodeLogic({ query: props.query, key, cachedResults: props.cachedResults, loadPriority, onData })
     const { response, responseLoading } = useValues(logic)
 
-    if (responseLoading) {
-        return (
-            <div className="w-full flex flex-col items-center text-2xl">
-                <Spinner />
-            </div>
-        )
-    }
-
-    if (!response) {
-        return null
-    }
-
     const webOverviewQueryResponse = response as WebOverviewQueryResponse | undefined
 
     const samplingRate = webOverviewQueryResponse?.samplingRate
 
     return (
         <>
-            <EvenlyDistributedRows className="w-full gap-x-2 gap-y-8" minWidthRems={8}>
```

Comment:
I like the background but I'd still do this with EvenlyDistributedRows, so that if there's space for 4 items they still get split 3-2 rather than 4-1

<img width="715" alt="Screenshot 2024-02-15 at 11 47 31" src="https://github.com/PostHog/posthog/assets/2056078/14415026-3b73-4cab-8742-b4d16b961fa8">